core/src/epicli/data/common/ansible/playbooks/roles/upgrade/tasks/verify.yml

```diff
@@ -1,46 +1,47 @@
 ---
-- name: Get cluster version
+- name: verify | Get cluster version
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl version --short | grep -i server
   changed_when: false
   register: kubectl_cluster_version
 
-- name: Get kubectl version
+- name: verify | Verify cluster version
+  assert:
+    that: "'{{ version }}' in kubectl_cluster_version.stdout"
+
+- name: verify | Get kubectl version
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl version --client --short | awk '{print $3}'
   changed_when: false
   register: kubectl_client_version
 
-- name: Get kubeadm version
+- name: verify | Verify kubectl version
+  assert:
+    that: "'{{ version }}' in kubectl_client_version.stdout"
+
+- name: verify | Get kubeadm version
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubeadm version -o short
   changed_when: false
   register: kubeadm_version
 
-- name: Get node version
+- name: verify | Verify kubeadm version
+  assert:
+    that: "'{{ version }}' in kubeadm_version.stdout"
+
+- name: verify | Verify node version
   environment:
     KUBECONFIG: /home/{{ admin_user.name }}/.kube/config
   shell: kubectl get nodes {{ inventory_hostname }} -o wide | awk '{print $2" "$5}'
   changed_when: false
   register: get_node_status
+  until: version in get_node_status.stdout
+  retries: 30 # 1min
```

Comment:
Won't be those comments misleading if someone will forget to change number of minutes after changing retries?